### PR TITLE
Split k8s bazel postsubmit in two.

### PR DESCRIPTION
**Pull request by spxtr**
In preparation for making an e2e job, I need the build job to run separate from the test job.

### CODE REVIEW

File: prow/postsubmit.yaml [hunk 1]
```diff
@@ -14,6 +14,17 @@ kubernetes/kubernetes:
...
         privileged: true
+  run_after_success:
+  - name: ci-kubernetes-bazel-test
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/bazelbuild:0.8
+        args:
+        - "--branch=$(PULL_REFS)"
+        - "--git-cache=/root/.cache/git"
+        - "--clean"
```
Comment (krzyzacy) [inline prow/postsubmit.yaml]: don't want to upload the log?

Comment (spxtr) [inline prow/postsubmit.yaml]: `--clean` to bootstrap means to do `git clean` before running tests.

Comment (spxtr) [inline prow/postsubmit.yaml]: Yes, fixed.

**Pull request merged**